forge.config.js

```diff
@@ -40,19 +43,32 @@ module.exports = {
     icon: getIcon(),
     asar: true,
     extraResource: getExtraResource(),
+    // osxSign: {
+    //   identity: 'Developer ID Application: Felix Rieseberg (LT94ZKYDCJ)',
+    //   'hardened-runtime': true,
+    //   entitlements: 'static/entitlements.plist',
+    //   'entitlements-inherit': 'static/entitlements.plist',
+    //   'signature-flags': 'library',
+    // },
+    // osxNotarize: {
+    //   appleId: 'felix@felix.fun',
+    //   appleIdPassword: 'my-apple-id-password',
+    // },
```

Comment:
Do we need this? If this is going to be needed in the future, let's leave it

Reply:
Yes we need it for MAC. and I'm waiting for the Signing Code